Source/WebKit/UIProcess/API/Cocoa/WKContentRuleListStore.mm

```diff
@@ -151,6 +151,54 @@ - (void)removeContentRuleListForIdentifier:(NSString *)identifier completionHand
 
 @implementation WKContentRuleListStore (WKPrivate)
 
+- (void)_compileContentRuleListFileAtPath:(NSString *)path withIdentifier:(NSString *)identifier encodedContentRuleList:(NSString *)encodedContentRuleList completionHandler:(void (^)(WKContentRuleList *, NSError *))completionHandler
```

Comment:
Looking at your use of this, we don't actually need any ObjC SPI for this at all.  Just call API::ContentRuleListStore functions directly.  Can it be tested through WebExtension SPI/API instead of the API test you added?